source/upgrade.txt

```diff
@@ -66,20 +66,97 @@ and upgrade versions.
   If you're upgrading the {+driver-short+} from version 3.0 to version 3.10,
   address all breaking changes listed for versions 3.1 to 3.10, if any.
 
+Types of Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion
```
Suggest merging this section into a continuation of the "Breaking Changes" section and/or consider making both subsections of the "Overview" section such that the ToC looks something like:

> On this page
> - Overview
>     - "This page describes..."
>     - "This page describes..."
> - Breaking Changes
>     - "A breaking change is..."
>     - "New versions of the C++ driver can introduce..."
>     - Version 4.0
>         - ...
>     - Version 3.11
>     - ...

or (my preference):

> On this page
> - Overview
>     - "This page describes..."
>     - "A breaking change is..."
>     - "New versions of the C++ driver can introduce..."
> - Breaking Changes
>     - Version 4.0
>         - ...
>     - Version 3.11
>     - ...